csgo_bot.py

```diff
@@ -102,6 +100,51 @@ def list_mix(message):
     items = db.get_items()
     bot.send_message(chat_id, items, parse_mode='Markdown')
 
+    
+	
+	
+@custom_group_only
+def get_javigon(message):
+	try:
+		chat_id = message.chat.id
+		markup = types.ReplyKeyboardMarkup(row_width=2)
+		itembtn1 = types.KeyboardButton('/sinDuda')
+		itembtn2 = types.KeyboardButton('/pollon')
+		markup.add(itembtn1, itembtn2)
+		bot.send_message(chat_id, "Elige tu audio:", reply_markup=markup)
+		
+	except:
+		print("Oops!Try again...")
+	
+	
+@custom_group_only
+def get_sinDuda(message):
+	try:
+		chat_id = message.chat.id
+		#print("eliminando teclado")	
+		markup = types.ReplyKeyboardRemove(selective=False)
+		bot.send_message(chat_id, "ok reproduciendo sinDuda", reply_markup=markup)
+		bot.send_audio(chat_id=chat_id, audio=open('javigon_sinDuda_audio.ogg', 'rb'))
+		bot.send_message(chat_id, "ok reproduciendo sinDuda", reply_markup=markup)
```

Comment:
Why are you sending the same message twice?

Reply:
solved